sktwocnt/lib/twocnt.f90

```diff
@@ -253,7 +253,7 @@ subroutine get_twocenter_integrals(inp, imap, skham, skover)
       call xc_f03_func_init(xcfunc_c, XC_GGA_C_PBE, XC_UNPOLARIZED)
     elseif (inp%iXC == 7) then
```

Comment:
Any reason these are suddenly numerical values instead of from an enum() ?

Reply:
No good reason, I just overlooked those occurrences when cleaning up, it is on my to-do list.